scripts/Interface/ui/sans_isis/sans_data_processor_gui.py

```diff
@@ -372,6 +374,10 @@ def _data_changed(self):
     def _instrument_changed(self):
         self._call_settings_listeners(lambda listener: listener.on_instrument_changed())
 
+    def _on_help_button_clicked(self):
```

Comment:
From within mantidplot, you should try this first `pymantidplot.proxies.showCustomInterfaceHelp('ISIS SANS v2')`